src/common/Filters.js

```diff
@@ -121,7 +121,7 @@ const InitializeDateValues = () => {
   var dateFormat = require("dateformat");
 
   const startDate = new Date().setFullYear(new Date().getFullYear() - 1);
-  const endDate = new Date().setMonth(new Date().getMonth() + 1);
+  const endDate = new Date().setMonth(new Date().getMonth());
 
```

Comment:
Does anyone know why we added this +1 to the month? It was forcing a future date search which is incorrect.

example: 4/1/2020, 5/1/2021 - this doesnt seem right since news stories cant happen yet so why search for a month in the future. 

Anyone know a reason why this was done or is it a carryover from a different time?

Reply:
I think the get month is zero based meaning January = 0 February = 1 so the +1 was to make it the correct date. But I think maybe that has been corrected in the date functions? Removing this +1 is selecting the correct month now so I think maybe this was just recently updated